.github/workflows/integration_tests_deployment.yml

```diff
@@ -0,0 +1,68 @@
+name: Integration Tests Deployment
+on:
+  workflow_call:
+    inputs:
+      Environment:
```

Comment:
Usually these GitHub action inputs are lowercased and kebab cased. Any reason to keep the first letter uppercased?

Reply:
updated